src/modules/feedback/components/mapFeedback/MapFeedbackPanel.js

```diff
@@ -306,14 +316,16 @@ export class MapFeedbackPanel extends MvuElement {
 		return `${this._configService.getValueAsPath('FRONTEND_URL')}?${decodeURIComponent(iframeParams.toString())}`;
 	}
 
-	/**
-	 * Registers a callback function which will be called when the form was submitted successfully.
-	 * @type {Function}
-	 */
 	set onSubmit(callback) {
 		this._onSubmit = callback;
 	}
 
+	set center(value) {
+		if (isCoordinate(value)) {
+			this.signal(Update_Center, value);
+		}
+	}
+
```

Comment:
Would suggest removing this check. (In `set onSubmit` we also do not check if it is a function)